sdk/data/azappconfig/CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 1.1.0 (Unreleased)
 
 ### Features Added
-
+* Added support for `Snapshots`.
```

Comment:
If you've got a good product page link describing snapshots you can link it here. 

Personally, it seems like a big feature to me and something people would want to jump on.